Dockerfile

```diff
@@ -56,12 +57,14 @@ RUN apt-get update && \
 	&& chown -R ${GUID}:${GUID} /home/${USER} \
 	&& echo 'ff ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers \
 	&& echo 'Defaults !requiretty' >> /etc/sudoers \
-	&& echo root:wscef | chpasswd \
-	# Cleanup
-	&& apt remove --purge -y wget \
-	&& apt autoremove -y \
+	&& echo root:wscef | chpasswd
+
+RUN apt -y install /src/GBPCEFwr64.deb || true
```

Comment:
The warsaw install fine on build? maybe not, I know it generates a hash and because this step is in startup file and not recommended if I push for dockerhub more layer... I need a way to reduce this

Reply:
Humn, I did not know that, probably the warsaw idea it to have a different hash for each installation... but yes, it works as is. I will move back warsaw install back to startup.sh

Reply:
#2